src/gettingstarted/cli.md

```diff
@@ -75,6 +75,54 @@ Options:
  --shell (-s)          Launch the shell
 ```
 
+### CLI features
+
+Additional settings to control the operation of the `platform` CLI can be managed in the configuration file (See `.platform/local/project.yaml`) or environment variables. See the [`platformsh-cli` `README` for details](https://github.com/platformsh/platformsh-cli/blob/master/README.md#usage). 
+
+#### Auto-selecting your project
+
+When your shells working directory is inside a local checkout of your project repository, the `platform` CLI tool will autodetect your project ID and environment, so you don't need to list them as parameters each time.
+
+Eg, in your home directory you need to provide the project ID as an argument each time:
+
+```bash
+~ $ platform project:info --project=acdefghijkl --environment=staging
+```
+You can instead get the same result with just:
+```bash
+~ $ cd myproject
+myproject $ project:info
+```
+
+You can also set a preferred project ID with the environment variables `PLATFORM_PROJECT`, `PLATFORM_BRANCH` and `PLATFORM_APPLICATION_NAME`..
+
+```bash
+export PLATFORM_PROJECT=acdefghijkl;
+export PLATFORM_BRANCH=staging;
+project:info
+```
+
+### Autocomplete on the commandline
+
+__When installed properly__\* the `platform` CLI tool provides tab autocompletion for commands, options, and even some values (your projects, valid regions). eg:
```

Comment:
Probably doesn't need the emphasis, maybe just a link to the installation instructions above:

[When in installed properly](/gettingstarted/cli.html#installation),

or 

[Once installed](/gettingstarted/cli.html#installation),